db_service/migrations.go

```diff
@@ -212,11 +212,11 @@ func RunMigrations(db *gorm.DB) error {
 
 	// if we've run any migrations before, we should have a migrations table, so find the last one we ran
 	if db.HasTable("migrations") {
-		lastMigration := models.Migration{}
-		if err := db.Order("created_at desc").First(&lastMigration).Error; err != nil {
+		var storedMigrations []models.Migration
+		if err := db.Order("id desc").Find(&storedMigrations).Error; err != nil {
```

Comment:
Looks like it is [auto incremented](https://github.com/GoogleCloudPlatform/gcp-service-broker/blob/master/db_service/migrations.go#L93). Recommend changing this to sort on `migraqtion_id`